app/src/main/kotlin/com/wire/android/notification/WireFirebaseMessagingService.kt

```diff
@@ -22,13 +22,15 @@ class WireFirebaseMessagingService : FirebaseMessagingService() {
 
     override fun onNewToken(p0: String) {
         super.onNewToken(p0)
-        coreLogic.getAuthenticationScope().saveNotificationToken(p0, "GCM").let { result ->
+        coreLogic.globalScope {
+            saveNotificationToken(p0, "GCM")
+        }.let { result ->
             when (result) {
                 is SaveNotificationTokenUseCase.Result.Failure.Generic -> {
                     appLogger.e("token registration has an issue : ${result.failure} ")
 
                 }
-                is SaveNotificationTokenUseCase.Result.Success -> {
+                SaveNotificationTokenUseCase.Result.Success -> {
```

Comment:
is there any reason to remove this `is` and leave the other one?

Reply:
`Success` is an object not a class, so it is not really needed I can add it if you want

Reply:
I missed that, thanks